### PR TITLE
config: add group and window options for locksmith

### PR DESCRIPTION
**Pull request by crawford**

### CODE REVIEW

**Comment by crawford** [comment]
Fixes https://github.com/coreos/bugs/issues/818 and https://github.com/coreos/bugs/issues/832.

**Comment by jonboulle** [comment]
Test for the regexen?

**Comment by crawford** [comment]
prolly

**Comment by philips** [comment]
Gah, wait, what, why are the env variables REBOOT_WINDOW_START and REBOOT_WINDOW_LENGTH not prefixed by LOCKSMITHD?

**Comment by philips** [comment]
Test the regex and we need to fix those env variables and do a new locksmith release @mischief

**Comment by philips** [comment]
https://github.com/coreos/locksmith/issues/90

**Comment by crawford** [comment]
Added tests

File: config/locksmith_test.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
+/*
```
Comment (jonboulle) [inline config/locksmith_test.go]: line comments + 2015 guy

**Comment by jonboulle** [comment]
LGTM after comment

**Pull request merged**